src/Number.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace FOP\Number;
```

Comment:
I would rather use `FriendsOfPhp\Number`. As far as I see other packages in this org don't use FOP-like top namespace at all, but I believe in this case it's better to add something, because standalone `Number` wouldn't be unique enough.

Reply:
Yeah I agree that it becomes kinda clunky to have to import `use VendorName\Number\Number;` as the class is there twice. But I also think it's kinda what we have to do as it's just the way namespaces usually are.

I'm assuming there could become conflicts if we simply used `FriendsOfPhp` as the namespace like this: `use FriendsOfPhp\Number;`, because what if a future package also uses that namespace?

Reply:
Duplicated `Number` does not bother me, because first one is sub-namespace and second one is a class name, it's OK IMHO. I was talking about `namespace Number\Number;` vs `namespace FriendsOfPhp\Number\Number;` 🙂. In terms of packages like [Pickle](https://github.com/FriendsOfPHP/pickle) it makes sense to use simple `namespace Pickle;`, but `namespace Number;` is not good enough 🙂.

Reply:
Ah got it. Makes total sense! 🙂